### ISSUE TITLE
Error withdrawing

### ISSUE THREAD

**Issue by willadamskeane** [jaggedsoft/node-binance-api#106]
I've intermittently been getting a "Withdraw address error" when trying to withdraw XLM using the API. I'm sure the address is correct, because it sometimes works.

Deleting the address from saved withdrawal addresses on the website seems to fix the problem, but of course this introduces quite a bottleneck when trying to automate the process of trading and withdrawing on Binance.

I understand this node implementation of the API is probably not responsible for the problem but I'm wondering if anyone has any insight into this bug?

**Comment by jaggedsoft** [comment]
I'll do some testing with this, I haven't tested it since they first came out with this feature and I'm certain some things have changed

**Comment by jaggedsoft** [comment]
It's kind of silly to me how this system works, name is described in their documentation as not being a mandatory parameter, but it actually is. And it appears you may run into this problem if you're submitting a withdraw with a `name` that has already been used before.
So, starting next version you can change the 'name' parameter at the end, I suppose this fix is only a kludge, but it should allow you to append a timestamp to it in order to make it unique
```js
        withdraw: function(asset, address, amount, addressTag = false, callback = false, name = 'API Withdraw') {
            let params = {asset, address, amount, name};
            if ( addressTag ) params.addressTag = addressTag;
            signedRequest(wapi+'v3/withdraw.html', params, callback, 'POST');
        },
```

**Comment by willadamskeane** [comment]
Thank you for clarifying, I was having a tough time pinpointing the root of that error message. It sounds like appending a timestamp to the withdrawal address name should avoid this problem.

Unfortunately, it may run into another issue that I've seen, since Binance will throw an error if you have more than 20 withdrawal addresses saved. Again, deleting these addresses manually from the website fixes the problem but that really isn't much of a solution.

**Comment by jaggedsoft** [comment]
One thing I haven't tested is sending a blank name parameter. Probably won't do much, but worth testing

**Comment by vikasprogrammer** [comment]
Sending blank name doesn't solve it either. We really need a workaround for this issue.

**Comment by sulistyo** [comment]
Does this ever get solved? what was the solution? appending timestamp does not solve it for me @jaggedsoft @vikasprogrammer @willadamskeane

**Comment by vikasprogrammer** [comment]
This is what I found, 

when withdrawing via Binance own Web UI, it does send a empty "name" param in the XHR request but it doesn't save the address automatically (unless you choose "New address" and specify a label). 

when withdrawing via API, "name" param is mandatory and sending blank does not work. I tried to send " " (space) as name param value but it saved that address as label "  ", lol. 

This is clearly a bug from Binance, I have sent an email to them specifying this problem. If you guys can send emails to them as well, this will become a must-solve issue for them.
 
We really need a solution for this. Right now, I have sniffed XHR request for listing withdrawal addresses and how to delete them. I am sending these requests via command line to delete the addresses programmatically, not an ideal solution, of course.

Hope this helped.
